SIS_Images_Scraper.py

```diff
@@ -0,0 +1,204 @@
+import getpass
+import urllib
+import os
+from selenium.webdriver.support.ui import Select
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+
+# Login to SIS
+def login(driver):
+    # Get User ID and PIN
+    user_id = getpass.getpass("User ID: ")
```

Comment:
Does this really need to be a getpass instance given how many other things we (at RPI) type our RIN into (and is probably stored as plaintext anyway). I've always hated how I couldn't see my RIN on this screen as well. Probably should just call in 'RIN' as well given how RPI specific SIS is.